Documents-Endpoint.md

```diff
@@ -196,7 +196,7 @@ Retrieve the full document bgu;11;2029
 | Key | Value | 
 | --- | ----- |
 | Content-Type | Content-Type: application/tei+xml |
-| Link | </dts/api/navigation/?id=bgu;11;2029>; rel="contents", </dts/api/collection/?id=bgu;11;2029>; rel="collection" | 
+| Link | </dts/api/navigation/?id=bgu;11;2029>; rel="contents", </dts/api/collections/?id=bgu;11;2029>; rel="collections" | 
```

Comment:
rel="collections" should remain rel="collection" based on the Link Relation Registry (See main comment).